### DISCUSSION TITLE
In the scenarios of kafka cluster and oap cluster, the monitoring metric cpm value is decreasing.

### DISCUSSION THREAD

**Discussion by pw151294** [apache/skywalking#11927] [Q&A]
Discription
1）Changing the way agents report data to Skywalking from GRPC to Kafka cluster mode.
2）OAP is in cluster mode with 3 nodes:
OAP node 172.31.91.218 role Aggregator
OAP node 172.30.15.183 role Receiver
OAP node 172.30.91.50 role Receiver
3）Kafka cluster mode with 3 nodes: 172.31.200.52:9092, 172.31.200.53:9092, 172.31.200.59:9092

Main Issues:
1）When sending 1000 requests to the monitored service, the cpm value displayed on the Skywalking-ui monitoring page is 1000 when data is reported in GRPC mode, and it is 350 when reported in Kafka cluster mode, significantly lower than 1000.
2）Modify the agent's configuration file of the monitored service, add the following configuration:
plugin.kafka.namespace=oap
Modify the application.yml configuration file of the oap node 172.31.91.218 in the kafka-fetcher module's namespace parameter

Question:
In the scenario of using a Kafka cluster to receive agent-reported data in an OAP cluster environment, why is it necessary to limit the agent data reporting to specific OAP nodes to ensure the correctness of cpm? Why does the cpm value decrease when using the default empty string as the namespace configuration?

**Comment by wu-sheng** [comment]
Duplicate https://github.com/apache/skywalking/discussions/11928. Don't submit repeated discussion again.
